### ISSUE TITLE
TTS pauses in the middle of speech if screen is off

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#2145]
On my new mobile phone it seems like AB is forcefully put to sleep during TTS unles app preference -> battery -> unrestricted is applied. Need to check if there's something we can do. If not anything else, should instruct user to switch that option manually.

**Comment by RuggMatt** [comment]
I made a small config change to the manifest. Can you see if it helps?

**Comment by tuomas2** [comment]
I'll try, thanks!

**Comment by tuomas2** [comment]
Yes, it seems to work. Now AB is set to "optimized" and it still does not break speaking.

**Comment by tuomas2** [comment]
Happened again
